src/partition/ReceivedPartition.cpp

```diff
@@ -426,11 +420,10 @@ void ReceivedPartition::compareBoundingBoxes()
 
     // send connectionMap to other master
     m2n().getMasterCommunication()->send(connectedRanksList, 0);
-    if (not connectionMap.empty()) {
-      com::CommunicateBoundingBox(m2n().getMasterCommunication()).sendConnectionMap(connectionMap, 0);
-    } else {
-      PRECICE_ERROR("This participant seems to have no mesh partitions for mesh " << _mesh->getName() << " at the coupling interface.");
-    }
+    PRECICE_CHECK(not connectionMap.empty(), "The mesh \"" << _mesh->getName() << "\" of this participant has no partitions at the coupling interface. "
+                                                                                  "Check that both mapped meshes are describing the same geometry. "
+                                                                                  "If you deal with very coarse meshes, consider increasing the safety-factor in the <use-mesh /> tag.");
```

Comment:
```suggestion
                                                                                  "If you deal with very different mesh resolutions, consider increasing the safety-factor in the <use-mesh /> tag.");
```
Just being coarse is no problem I guess?

Reply:
Your version is what I was looking for :wink: